### DISCUSSION TITLE
Compatibility and Functionality

### DISCUSSION THREAD

**Discussion by Sash-07** [snipe/snipe-it#9472] [Q&A]
Hi guys, please can you confirm if the below tablet will allow us to scan barcodes and have full functionality of the Snipe IT app

![Samsung Tablet](https://user-images.githubusercontent.com/83006774/115692012-df16d180-a35e-11eb-94a7-3ab774a5a219.png)

![Samsung specs](https://user-images.githubusercontent.com/83006774/115692103-f8b81900-a35e-11eb-91d9-f6253b346502.png)

![Samsung specs 2](https://user-images.githubusercontent.com/83006774/115692174-08376200-a35f-11eb-81c1-a9431f18b465.png)

thanks

**Comment by csayre** [comment]
The webpage should load on that and you can take and upload images. But Snipeit (to my knowledge) does not have the ability to scan a barcode and look up an asset. 

Maybe you can use an app that will let you scan a barcode and then paste the text into a web page.
